package.json

```diff
@@ -90,6 +94,7 @@
     "clean-webpack-plugin": "0.1.10",
     "compression": "1.6.2",
     "compression-webpack-plugin": "0.3.2",
+    "concat-map": "0.0.1",
```

Comment:
Why is this added?

Reply:
Left over from previous try. i'll get rid of them